organization/app/organization/[id]/layout.tsx

```
@@ -1,12 +1,326 @@
+                    initial={{ opacity: 0 }}
+                    animate={{ opacity: 1 }}
+                    exit={{ opacity: 0 }}
+                    className="flex items-center"
+                  >
+                    <span className="font-bold text-xl text-blue-600">TurfMania</span>
+                  </motion.div>
+                )}
+                <button 
+                  onClick={() => setIsCollapsed(!isCollapsed)}
+                  className="p-2 rounded-lg hover:bg-gray-100 transition-colors"
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)